pkg/server/options/options.go

```diff
@@ -287,6 +287,15 @@ func (o *Options) Complete(ctx context.Context, rootDir string) (*CompletedOptio
 		}
 	}
 
+	// ExternalAddress defaults to the default interface if unset and
```

Comment:
Can you please re-phrase this comment? It reads a bit awkwardly.

Reply:
```suggestion
	// ExternalAddress is the address used e.g. when generating kubeconfigs
	// or the virtual workspace URLs. It defaults to the default interface.
	// If the user provides BindAddress but not ExternalAddress the generated
	// kubeconfigs and URLs might not be valid.
	// Hence ExternalAddress is set here to the value of BindAddress only if
	// BindAddress is set but not ExternalAddress.
	// If neither are provided then the server listens on all interfaces but advertises
	// only the default interface.
```

Oh yeah, you're right. I think I had squashed two different comments together and that's why its so rough.